### PR TITLE
caffeine, caffeinate: add pages

### LINKED ISSUE #8138: page modification: caffeine, caffeinate
Currently says "Prevent macOS from sleeping"

But is also available for GNU/Linux

Perhaps it should be moved from pages/osx to pages/common, or given its own entry in pages/linux. :)

### PR DESCRIPTION
**Pull request by th0ughtcrime**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

Closes #8138

### CODE REVIEW

File: pages/linux/caffeinate.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# caffeinate
+
+> Prevent Linux from sleeping.
```
Comment (kbdharun) [inline pages/linux/caffeinate.md:3]:
```suggestion
> Prevent Desktop from sleeping.
```

File: pages/linux/caffeinate.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+> See also: `caffeine`.
+> More information: <https://manned.org/caffeinate.1>.
```
Comment (kbdharun) [inline pages/linux/caffeinate.md:5]:
```suggestion
> More information: <https://manned.org/caffeinate>.
```

File: pages/linux/caffeine.md [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# caffeine
+
+> Prevent Linux from sleeping.
+> See also: `caffeinate`.
+> More information: <https://manned.org/caffeine.1>.
```
Comment (kbdharun) [inline pages/linux/caffeine.md:5]:
```suggestion
> More information: <https://manned.org/caffeine>.
```

File: pages/linux/caffeine.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+# caffeine
+
+> Prevent Linux from sleeping.
```
Comment (kbdharun) [inline pages/linux/caffeine.md:3]:
```suggestion
> Prevent Desktop from sleeping.
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

**Comment by th0ughtcrime** [comment]
@kbdharun Thanks :) I wasn't sure about the description so I made it reflect the MacOS page.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages/linux/caffeinate.md [hunk 5]
```diff
@@ -0,0 +1,9 @@
+# caffeinate
+
+> Prevent Desktop from sleeping.
```
Comment (CleanMachine1) [inline pages/linux/caffeinate.md:3]:
```suggestion
> Prevent the desktop from sleeping while a command is running.
```

Comment (kbdharun) [reply pages/linux/caffeinate.md:3]: This looks better

Comment (th0ughtcrime) [reply pages/linux/caffeinate.md]: Thanks guys. Have another look :)

**Pull request closed without merging**